Runnect-iOS/Runnect-iOS/Presentation/CourseStorage/Views/CourseListView/PrivateCourseListView.swift

```diff
@@ -8,14 +8,31 @@
 import UIKit
 import Combine
 
+// protocol PrivateCourseListViewDelegate: AnyObject {
+//    func deleteCourseButtonDidTap()
+//}
```

Comment:
protocol-delegate 패턴을 이용해 상위 VC인 CourseStorageVC에 선택된 코스들의 id 어레이를 넘겨주는 방식으로 해야할 것 같습니다.

```suggestion
protocol PrivateCourseListViewDelegate: AnyObject {
    func deleteCourseButtonDidTap(courseIdsToDelete: [Int])
}
```